include/mxnet/base.h

```diff
@@ -143,7 +143,8 @@ struct Context {
   enum DeviceType {
     kCPU = cpu::kDevMask,
     kGPU = gpu::kDevMask,
-    kCPUPinned = 3
+    kCPUPinned = 3,
+    kCPUShared = 5,
```

Comment:
Curious, why is this needed?

Reply:
For creating shared memory from frontend